### DISCUSSION TITLE
Keep getting rejected for Global Campus

### DISCUSSION THREAD

**Discussion by TheToastyAbyss** [community/community#56599] [GitHub Education] [answered]
### Hi everyone,

Every time I submit my application I keep getting rejected.

The things I have tried:
I have updated my GitHub profile with my bio and full name.
I have tried uploading the documents via a picture from my computer and converting the pdf to a jpeg file

I've tried going through the forum and don't have any other documents I can submit besides my active transcript. My student ID does not have any dates on it.

Thank you for taking the time to read this and help me

Here are the submissions.

Rejected Submitted May 28, 2023
Unfortunately, we weren't able to approve your educational discount request for [@TheToastyAbyss](https://github.com/TheToastyAbyss).

What happened?
The item you uploaded is insufficient to demonstrate your current academic status.
Please see suggestions.
Suggested proof to expedite your approval:

You are unlikely to be verified until you have completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with your full name as it appears in your academic affiliation document plus a short bio.

Please consider using your device camera to submit academic affiliation documents. Uploaded images are more easily manipulated and are therefore less trustworthy.

Rejected Submitted May 28, 2023
Unfortunately, we weren't able to approve your educational discount request for [@TheToastyAbyss](https://github.com/TheToastyAbyss).

What happened?
The item you uploaded is insufficient to demonstrate your current academic status.
Please see suggestions.
Suggested proof to expedite your approval:

You are unlikely to be verified until you have completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with your full name as it appears in your academic affiliation document plus a short bio.

Rejected Submitted May 28, 2023
Unfortunately, we weren't able to approve your educational discount request for [@TheToastyAbyss](https://github.com/TheToastyAbyss).

What happened?
The item you uploaded is insufficient to demonstrate your current academic status.
Please see suggestions.
Suggested proof to expedite your approval:

You are unlikely to be verified until you have completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with your full name as it appears in your academic affiliation document plus a short bio.

Rejected Submitted May 28, 2023
Unfortunately, we weren't able to approve your educational discount request for [@TheToastyAbyss](https://github.com/TheToastyAbyss).

What happened?
The item you uploaded is insufficient to demonstrate your current academic status.
Please see suggestions.
Suggested proof to expedite your approval:

You are unlikely to be verified until you have completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with your full name as it appears in your academic affiliation document plus a short bio.

Please consider using your device camera to submit academic affiliation documents. Uploaded images are more easily manipulated and are therefore less trustworthy.

Best Regards,

John Pereira

**Comment by jge162** [comment, marked as answer]
Are you using your school email as your primary for your GitHub account? If not change it to school email. Show current transcripts as submission docs.

**Reply by TheToastyAbyss** [reply]
I just changed my primary email and used my current transcripts as a doc. The application is now pending, thanks for the help! I will post down below if it does not work.

**Reply by TheToastyAbyss** [reply]
Update: It worked, Thank you so much!

**Reply by jge162** [reply]
Glad to hear. Enjoy
